McpPlugin/test_eps6800_mcp.ps1

```
@@ -0,0 +1,492 @@
+	} while (-not $recordingStatus.paused -and $recordingStatus.program_counter -eq $recordingBaseline.program_counter -and (Get-Date) -lt $recordingDeadline)
+	$null = Invoke-McpTool 30 "pause"
+	$null = Invoke-McpTool 31 "stop_call_recording"
+	$calls = Invoke-McpTool 32 "list_function_calls"
+	Assert-True ($calls.calls.Count -gt 0) "EPS6800 Call Analysis did not receive call hooks."
+
+	# HP300S+ reserves model button codes FE/FF for RESET and independent ON.
+	# Verify the model parser and desktop Keyboard path, not only the core API.
+	$null = Invoke-McpTool 61 "step_into"
+	$beforeResetButton = Wait-StatusPaused 62
+	Assert-True ($beforeResetButton.program_counter -ne 0) "Precondition for RESET button test was not reached."
+	$null = Invoke-McpTool 63 "keyboard_code" @{ code = 0xFE; pressed = $true }
```

PSScriptAnalyzer (1.25.0)

[info] Cmdlet 'Invoke-McpTool' has positional parameter. Please use named parameters instead of positional parameters when calling a command.

(PSAvoidUsingPositionalParameters)